spec/rspec/mocks/nil_expectation_warning_spec.rb

```diff
@@ -24,6 +24,13 @@ module Mocks
         nil.foo
       end
 
+      it "raises an error when expectations on nil are disallowed" do
+        disallow_message_expectations_on_nil
+
+        expect { expect(nil).to receive(:foo)     }.to raise_error(StandardError)
```

Comment:
nit: particularly with such a generic error type, I'd typically do `raise_error(/foo/)` to make sure the error is a) relevant, and b) contains relevant debugging information (tests the interpolation)